alkira/helper.go

```diff
@@ -52,10 +52,14 @@ func expandSegmentOptions(in *schema.Set, m interface{}) (alkira.SegmentNameToZo
 			return nil, errors.New("segment_option fields cannot be nil")
```

Comment:
I am not sure if this is true... e.g. for service_pan, I think the zone to group mapping (now it's SegmentOptions) is optional. Not true anymore?

Reply:
I'll double check the docs and run a test to see. If not required will remove.